[Docs] Add why Vizro-AI section (#916)
mckinsey/vizro: vizro-ai/docs/pages/explanation/why-use-vizro-ai.md

# Why use Vizro-AI over ChatGPT?

Vizro-AI offers an easy way to turn natural language instructions into professional visualisations, like interactive charts and dashboards, without needing advanced coding skills.

> Similar to ChatGPT, Vizro-AI uses generative AI to understand and execute your instructions in English or any other language. However, there are few advantages of using Vizro-AI over ChatGPT.

Reviewer: ```suggestion
Like ChatGPT, Vizro-AI uses generative AI to understand and execute your instructions in English, or any other language. However, there are few advantages of using Vizro-AI over ChatGPT.
```

> - **Tailored for data visualisation**: Vizro-AI is specifically designed to create and design charts and dashboards.

Reviewer 2: I like those suggestions - I think they might both become individual points even

> - **Enforced design best practices automatically**: Using Vizro’s themes, your visualizations are built with best practices in mind from the start.

Reviewer: I would steer away from "enforced" since that sounds a bit draconian. Maybe "Design best practices applied automatically" ?

> - **Instant chart execution**: Vizro-AI generates and executes chart code automatically, allowing you to see the chart output immediately

Reviewer: Please replace "allowing" with "enabling"

# Why use Vizro-AI over ChatGPT?

Vizro-AI offers an easy way to turn natural language instructions into professional visualizations, like interactive charts and dashboards, without needing advanced coding skills.

Like ChatGPT, Vizro-AI uses generative AI to understand and execute your instructions in English, or any other language. However, there are few advantages of using Vizro-AI over ChatGPT.

> - **Tailored for data visualization**: Vizro-AI is specifically designed to create and design charts and dashboards.

Reviewer 3: ```suggestion
- **Tailored for data visualization**: Vizro-AI is specifically designed to create charts and dashboards with in-built beautiful visual design best practices.
```

I would rephrase to something like the above because Vizro-AI and Vizro overall don't assist in designing new charts and dashboards. Instead, they provide an opinionated design through Vizro, but they don't "help" in the sense of creating new designs, which I would partially understand from above

> - **Enforced design best practices automatically**: Using Vizro’s themes, your visualizations are built with best practices in mind from the start.

Reviewer 3: ```suggestion
- **Data visualization best practices applied**: Vizro’s chart theme ensures your charts follow many best practices, such as using color-blind friendly colors, maintaining a decluttered design, and automatically placing headers and footers.
```

> - **Prompt optimization**: Vizro-AI uses advanced prompt engineering to incorporate additional context and data-specific details, making your visualizations more accurate.

Reviewer 3: This seems too focused on the technical implementation rather than the user benefit. As a user, I'm more interested in how it provides data insights that complement my charts. So I think the emphasis should be on the added value rather than the technical details like prompt optimization?  Maybe something like below?

```suggestion
- **Data Insights**: Vizro-AI leverages advanced prompt engineering to integrate additional context and data-specific insights, enriching your visualizations with more comprehensive information and allowing for more effective data storytelling.
```
